tutorials/applications/Simulating Sequences with Errors and Noise.ipynb

```diff
@@ -0,0 +1,1129 @@
+{
```

Comment:
<p>Now that I'm seeing this here, it looks weird that we have no noise yet all the noise parameters appear defined here. Wouldn't it make more sense to show a given parameter if the respective noise type is part of the config? For example, only <code>Dephasing probability:</code>  if there is <code>depahsing</code>,  <code>Temperature</code>  if there is <code>doppler</code>,  etc...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Agreed, it'd look cleaner.</p>